public/views/index.html

```diff
@@ -14,6 +14,15 @@
 
     <base href="[[.AppSubUrl]]/" />
 
+    <script>
+      // NI fork: Ensure that grafana is loaded within an iframe unless it is running locally
+      if (window.self === window.top && window.location.hostname !== 'localhost') {
+        // Redirect to the iframe-hosted version of grafana, which is located at /dashboards rather than /dashboardhost
+        let newHref = window.location.href.replace(window.location.origin + '/dashboardhost', window.location.origin + '/dashboards');
+        window.location.replace(newHref);
```

Comment:
Should we be defensive and only do the window location change if `/dashboardhost` is actually present in the URL?
Maybe console.error or something a message if it's not there?